vendors/pc/boards/windows/ports/ota_pal_for_aws/ota_pal.c

```diff
@@ -53,43 +61,42 @@ static inline BaseType_t prvContextValidate( OtaFileContext_t* pFileContext )
 #define OTA_PAL_INT16_NEGATIVE_MASK    ( 1 << 15 )
 
 /* Size of buffer used in file operations on this platform (Windows). */
-#define OTA_PAL_WIN_BUF_SIZE ( ( size_t ) 4096UL )
+#define OTA_PAL_WIN_BUF_SIZE           ( ( size_t ) 4096UL )
 
 /* Attempt to create a new receive file for the file chunks as they come in. */
 
-OtaPalStatus_t otaPal_CreateFileForRx( OtaFileContext_t* const C )
+OtaPalStatus_t otaPal_CreateFileForRx( OtaFileContext_t * const C )
 {
     OtaPalMainStatus_t mainErr = OtaPalSuccess;
     OtaPalSubStatus_t subErr = 0;
-	int32_t lFileCloseResult;
+    int32_t lFileCloseResult;
 
     if( C != NULL )
     {
-        if ( C->pFilePath != NULL )
+        if( C->pFilePath != NULL )
         {
-			if ( C->pFile != NULL )
+            if( C->pFile != NULL )
             {
                 LogInfo( ( "File already open , closing first.\r\n" ) );
-				
-				lFileCloseResult = fclose( C->pFile ); /*lint !e482 !e586
-                                                        * Context file handle state is managed by this API. */
-				C->pFile = NULL;
 
-				if( 0 == lFileCloseResult )
-				{
-					LogInfo( ( "File closed.\r\n" ) );
+                lFileCloseResult = fclose( C->pFile ); /*lint !e482 !e586
```

Comment:
Is it required to close ? If the file is open can we use the same handle ?

Reply:
This was not a change I made so I can't answer this for sure without looking into it more. There is only a diff here because I ran uncrustify.

Lets address this in a separate PR since this would be a functional change unrelated to this PR.

Reply:
We agreed offline that this is not an issue so I'm resolving it.